### ISSUE TITLE
Extension panels repeatedly seeing didSeek=true when connected to a data source that does not support seeking (ws-protocol)

### ISSUE THREAD

**Issue by jhurliman** [foxglove/studio#4917]
1. Connect to `foxglove_bridge` and subscribe to at least two topics, one lightweight and one heavyweight (such as an image stream).
2. Observe 3D panel and Image panel repeatedly flashing/strobing

This is due to many frames coming in with `didSeek=true`. Any time a message is received on the WebSocket with a message timestamp less than the previous message timestamp, this happens:

```typescript
// If time goes backwards, increment lastSeekTime and discard unemitted messages from before
// the discontinuity. This prevents us from queueing an unbounded number of messages when
// servers loop over the same recorded data multiple times. However, for now the queue can
// still grow unboundedly in a live system if the listener is not processing messages (such
// as when the app is hidden/backgrounded).
if (this._currentTime && isLessThan(receiveTime, this._currentTime)) {
  ++this._lastSeekTime;
  this._parsedMessages = [];
}
```

**Comment by defunctzombie** [comment]
John: "I can tell you, I didn't seek. That's why it's marked as a bug"

**Comment by jhurliman** [comment]
Here's a visual demonstration:

https://user-images.githubusercontent.com/195374/205184152-bc0673b5-393d-4e65-b188-83867e74c389.mp4

**Comment by jtbandes** [comment]
Either duplicate of, or just blocked by, https://github.com/foxglove/studio/issues/3194
